test/test_prepared.py

```diff
@@ -0,0 +1,14 @@
+import psycopg
+import pytest
+
+def test_prepared_statement(bouncer):
+    bouncer.admin(f"set pool_mode=transaction")
+    prepared_query = "SELECT 1"
+    with bouncer.cur() as cur1:
+        with bouncer.cur() as cur2:
+            cur1.execute(prepared_query, prepare=True)
+            cur1.execute(prepared_query)
+            with cur2.connection.transaction():
+                cur2.execute("SELECT 2")
+                cur1.execute(prepared_query)
+
```

Comment:
Braindump of ideas of tests that should be added:
1. Eviction from prepared statement cache (by using a very small prepared_statement_cache_size)
2. Increasing prepared_statement_cache_size
3. Decreasing prepared_statement_cache_size
4. Setting prepared_statement_cache_size from non-zero to zero
5. Using multiple different prepared statements
6. Using the same prepared statement on different connections

Reply:
@knizhnik I updated the README on how to run tests, with install instructions using a virtual environment: https://github.com/pgbouncer/pgbouncer/blob/master/test/README.md#setting-up-python-dependencies-for-testing

Can you try those out and see if you can get the tests to work on your own machine? And if sou could you take on writing some of these tests? I implemented the last one of these today.

Reply:
Thank you for instructions.
Now everything works like a charm